client/my-sites/posts/post-share.jsx

```diff
@@ -130,6 +133,17 @@ const PostSharing = React.createClass( {
 			'has-connections': this.hasConnections()
 		} );
 
+		if ( ! this.props.planHasRepublicizeFeature ) {
+			return ( <div className="posts__post-share-wrapper">
+				<UpgradeNudge
```

Comment:
We should remove all instances of the `UpgradeNudge` in favor of the `Banner`, yes. 👍